project-base/docker/php-fpm/Dockerfile

```diff
@@ -84,10 +96,10 @@ FROM base as development
 USER root
 
 # allow overwriting UID and GID o the user "www-data" to help solve issues with permissions in mounted volumes
-# if the GID is already in use, we will assign GID 82 instead (82 is the standard uid/gid for "www-data" in Alpine)
+# if the GID is already in use, we will assign GID 33 instead (33 is the standard uid/gid for "www-data" in Debian)
 ARG www_data_uid
 ARG www_data_gid
-RUN if [ -n "$www_data_uid" ]; then deluser www-data && (addgroup -g $www_data_gid www-data || addgroup -g 82 www-data) && adduser -u $www_data_uid -D -S -G www-data www-data; fi;
+RUN if [ -n "$www_data_uid" ]; then deluser www-data && (addgroup --gid $www_data_gid www-data || addgroup --gid 33 www-data) && adduser --system --home /home/www-data --uid $www_data_uid --disabled-password --gid $www_data_gid www-data; fi;
```

Comment:
When gid is in use and www-data group is created with default 33 gid, you still create user with gid from argument. On Mac, the default gid of user is 20 (staff), but this gid is already in use by debian group dialout. Resulting in user www-data with group dialout and group www-data is unused.

Reply:
If I understand it correctly, the solution should be to add user using usermod to an existing group by name, not by ID. Can you please test it? Im not entirely sure.

Reply:
I've tried and user has group www-data, along with somw weird ones. It should be enough to create user with `group` option and do not modify at all. 
Suggestion is in files.